### PR TITLE
Devise

### PR DESCRIPTION
**Pull request by OmarMWarraich**
# Worthy Reviewer

## Kindly approve the fulfilled project requirements.

Remove current_user method in ApplicationController, devise will provide us with one.
    [Install and setup devise](https://github.com/heartcombo/devise#getting-started).
        Can register a new user.
        User logs in with a combination of email and password.
        Hashed password should be stored in the database.
        Ask for confirmation of email.
        Can reset password.
Styling

**_ScreenShots_**
<div style="width:100%;height:0;padding-bottom:100%;position:relative; display:flex">

</div>

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (mugishasam123) [inline app/models/user.rb]: - [x] KIndly try to add an extra layer of security by asking a user to confirm his or her email in the account creation to make your app more professional and secure. you can try to add `: confirmable` in devise  .you got this :+1:

**Review by mugishasam123 — CHANGES_REQUESTED** [review]
Hi @OmarMWarraich  :wave: , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
alt
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors :1st_place_medal: 
- Descriptive PR :rocket: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/user.rb (hunk 1 above)
Comment (OmarMWarraich) [reply app/models/user.rb]: @mugishasam123 Done.

**Review by mugishasam123 — APPROVED** [review]
Hi @OmarMWarraich  :wave: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors :heavy_check_mark: 
-  Descriptive PR summary :+1: 

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**